src/main/resources/templates/email/programme-created/v1.0.0.html

```diff
@@ -0,0 +1,77 @@
+  <p>Their pre-employment checks will include:</p><ul>
+  <li>Your right to work</li>
+  <li>If you have a criminal record</li>
+  <li>Employment history and references</li>
+  <li>Professional Registration and qualifications</li>
+  <li>Fitness to work</li>
+</ul>
+  <p><b><u>ACTIONs</u></b></p>
+  <p><b>TIS Self-Service (TSS)</b></p>
+  <p>TIS Self-Service (TSS) gives you access to your training information. You will need to use TSS to submit essential forms and sign training agreements and complete actions that are presented on the Action Summary.</p>
+  <th:block th:switch="${isRegistered}"
+  ><p th:case="true">
+    We have a registered account for you on TIS Self Service against the email address <b><span th:text="${email}"></span></b>. To access details for onboarding onto your programme and actions to complete please log in to TIS Self Service: <a href="https://trainee.tis.nhs.uk/home">https://trainee.tis.nhs.uk/home</a>
+  </p>
+    <p th:case="false">
+      We have no TIS Self Service account associated to your record. Please create one by clicking the link below.
+      <br/>
+      Please ensure you use the email address we have against our records <b><span th:text="${email}"></span></b>
+      <br/>
+      <a href="https://trainee.tis.nhs.uk/">https://trainee.tis.nhs.uk/</a>.
+    </p>
+  </th:block>
+  <p>If your email address is likely to change, or you need support signing up on TIS Self Service then please ensure that you contact us via <th:block th:switch="${contactHref}"
+  ><span th:case="url"><a th:href="${localOfficeContact}"><span th:text="${localOfficeContact}"></span></a></span
+  ><span th:case="email"><a th:href="'mailto:' + ${localOfficeContact}"><span th:text="${localOfficeContact}"></span></a></span
+  ><span th:case="*"><span th:text="${localOfficeContact}"></span></span
+  ></th:block> or see our <a href="https://tis-support.hee.nhs.uk/trainees/changes-to-account/">FAQ</a> for details.
+  </p>
+  <p>Once you access TIS Self Service, you will be presented with a set of actions and notifications to read before relating to your training programme. Please complete the actions and read the notification as requested.</p>
+  <p>For more information regarding the onboarding process please visit our website : <a th:href="${localOfficeWebsite}"><span th:text="${localOfficeWebsite}"></span></a>.</p>
+  <p><b>We are currently piloting the onboarding of Doctors to new programme service. You may come across a series of questions about this within the TIS Self Service app. We appreciate your feedback to help us improve the process and make it easy for you to access the necessary information.</b></p>
+  <p>We look forward to welcoming you to <span th:text="${not #strings.isEmpty(localOfficeName)} ? ${localOfficeName} : _"></span></p>
```

Comment:
Needs a bit of rework to avoid a cut of sentence if `localOfficeName` not available (or assume that it must always be available and remove the conditional part)

Reply:
Standardised to 'your local deanery office' if missing, pending confirmation from the powers that be.